app/tasks/task6/form.html

```diff
@@ -0,0 +1,110 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Form</title>
+
+</head>
+<!-- Shift + Alt + F for formatting the code -->
+
+<body>
+    <h1>First form</h1>
+
+    <form action="#" method="post">
+        <fieldset>
+            <legend>Personal Info</legend>
+            <div>
+                <label for="name"> First Name:</label>
+                <input type="text" id="fname" name="name" placeholder="Enter your  first name" required>
+            </div>
+            <div>
+                <label for="name"> Last Name:</label>
+                <input type="text" id="lname" name="name" placeholder="Enter your last name" required>
+            </div>
+            <div>
+                <label for="email">Email:</label>
+                <input type="email" id="email" name="email" placeholder="Enter your email" required>
+            </div>
+            <div>
+                <label for="email">Password:</label>
+                <input type="password" id="Password" name="password" placeholder="Enter your password (min. 8 characters)" minlength="8" required>
+            </div>
+            <div>
+                <label for="phone">Phone:</label>
+                <input type="tel" id="phone" name="phone" placeholder="Enter your phone number" required>
+            </div>
+            <div>
+                <label for="adress">Address:</label>
+                <input type="address" id="adress" name="adress" placeholder="Enter your adress" required>
```

Comment:
input de tip adress nu exista. Cred ca ar trebui sa fie de tip text sau url daca merge vorba de o adresa web.

Reply:
fixed, changed to type="text"